tests/FSharp.Compiler.ComponentTests/ErrorMessages/ClassesTests.fs

```diff
@@ -542,4 +542,148 @@ type C() =
         |> withDiagnostics [
             (Error 855, Line 7, Col 19, Line 7, Col 21, "No abstract or interface member was found that corresponds to this override")
             (Error 855, Line 9, Col 19, Line 9, Col 21, "No abstract or interface member was found that corresponds to this override")
-        ]
\ No newline at end of file
+        ]
+
```

Comment:
The tests look good but feels like the test matrix can somehow be bigger.  

* Interfaces?
* Add more tests for generics?
* sealed methods?

These should all be ok but you know, just in case